OpenGL/Math/Matrix4.cs

```diff
@@ -32,12 +32,12 @@ public static Matrix4 Identity
 
         public static Matrix4 operator *(Matrix4 m, Matrix4 m2)
         {
-            Matrix4 m2t = m2.Transpose();
+            Matrix4 mt = m.Transpose();
```

Comment:
I'll leave a comment in the PR itself, but unfortunately this is a breaking change and existing behaviour is actually intentional.